### DISCUSSION TITLE
Heartfelt Thank You and Appreciation

### DISCUSSION THREAD

**Discussion by steve-hardman** [kubernetes/website#44592] [General]
Hi there!

I'm a student and have been exploring Kubernetes. I've been curious about how K8s project works, and I wanted to see if I could contribute in some way. That's when I found this website's repository almost the start of this year. My experience of contributing has been really positive, and I want to give a shout-out to the maintainer, @dipesh-rawat ,who is very active in issues, comments and pull requests. I've noticed positive comments and great reviews, and even though I made some typos in my PR (I promise to be better next year), @dipesh-rawat has been supportive. Sometimes, I've seen him providing the entire solution to an issue with exact line number to change and helping with GitHub commands to get around build errors or rebase message. Even though it might have been faster for him to do it himself, the intention was to help newcomers like me, and for that, I sincerely thank you!

Also, big thanks to @sftim for reviewing, helping with everyone's queries in the comments and more improtantly merging PRs. Your efforts are greatly appreciated, and it's been a pleasant experience.

You guys are AWESOME!

I hope it's okay to share this gratitude here on the Github issue. I couldn't resist expressing my gratitude at the end of the year.

THANK YOU!

Wishing you a Happy New Year🎉

**Comment by Gauravpadam** [comment]
+1 to the emotion! I've met some great people through this community. I can't name a few and miss others out, But everyone has played some part in making this journey possible

Happy new year everyone! 🎉
Onwards & Upwards

**Comment by sftim** [comment]
(I converted this from an issue to a discussion)

Thanks for the feedback!

**Comment by dipesh-rawat** [comment]
@steve-hardman Thank you so much for your kind words and for being part of the community! It's fantastic to hear about your positive experience and growth. Your contributions are valuable. Looking forward to seeing more of your work in the coming year!

**Comment by thockin** [comment]
It's far more common to get rude messages from entitled people who don't contribute, than it is to get something like this.  So thanks for the thanks, and I agree - there are a lot of awesome people here!

**Comment by sftim** [comment]
Feedback very much noted and appreciated!
